app/views/pages/home.html.erb

```diff
@@ -74,13 +73,14 @@
 </div>
 
 
-
-
-
-
-
-
-
-
+<script>
+  var typed = new Typed('#typing', {
+  strings: ['For my project I need','For my project I need a hammer','For my project I need a screwdriver','For my project I need a drill'],
+  typeSpeed: 40,
+  backSpeed: 75,
+  showCursor: true,
+  smartBackspace: true
+  });
+</script>
```

Comment:
I would put this in a separate file that I would create in `app/javascript/packs/components/init_typed.js`. wrap the logic in a function like
```js
// import the javascript package
import Typed from 'typed.js';
const initTypedJs = () => {
 new Typed('#typing', {
  strings: ['For my project I need','For my project I need a hammer','For my project I need a screwdriver','For my project I need a drill'],
  typeSpeed: 40,
  backSpeed: 75,
  showCursor: true,
  smartBackspace: true
  });
};

export = { initTypedJs };
```
and then import it in `app/javascript/packs/application.js` and call it.

Reply:
Done!